buildSrc/src/main/kotlin/com.workos.java.examples.java-common-conventions.gradle.kts

```diff
@@ -21,3 +21,10 @@ dependencies {
 tasks.test {
   useJUnitPlatform()
 }
+
+java {
+  toolchain {
+    // Determines JDK version used for compilation. See https://docs.gradle.org/current/userguide/toolchains.html
+    languageVersion.set(JavaLanguageVersion.of(8))
```

Comment:
This determines what JDK version is used for compilation, rather than just using whatever is currently installed on a developer's machine. It will also install the specified version of the JDK if it's not already which makes getting started much easier.

Reply:
When I run the d-sync app and click list directories I'm getting a couple errors in regard to "var" not being a recognized symbol. 
```shell
> Task :java-directory-sync-example:run
[main] INFO org.eclipse.jetty.util.log - Logging initialized @648ms to org.eclipse.jetty.util.log.Slf4jLog
[main] INFO io.javalin.Javalin - Static file handler added: StaticFileConfig(hostedPath=/, directory=src/resources, location=EXTERNAL, precompress=false, aliasCheck=null, headers={Cache-Control=max-age=0}, skipFileFunction=(javax.servlet.http.HttpServletRequest) -> kotlin.Boolean). File system location: 'src/resources'
[main] INFO io.javalin.Javalin - 
       __                      __ _            __ __
      / /____ _ _   __ ____ _ / /(_)____      / // /
 __  / // __ `/| | / // __ `// // // __ \    / // /_
/ /_/ // /_/ / | |/ // /_/ // // // / / /   /__  __/
\____/ \__,_/  |___/ \__,_//_//_//_/ /_/      /_/

          https://javalin.io/documentation

[main] INFO io.javalin.Javalin - Starting Javalin ...
[main] INFO io.javalin.Javalin - You are running Javalin 4.1.1 (released October 10, 2021).
[main] INFO io.javalin.Javalin - Listening on http://localhost:7001/
[main] INFO io.javalin.Javalin - Javalin started in 159ms \o/
[qtp418958713-13] WARN io.javalin.Javalin - Uncaught exception
gg.jte.TemplateException: Failed to compile template, error at directories.jte:4
/Users/adamwolfman/Documents/example_apps/java-example-applications/java-directory-sync-example/jte-classes/gg/jte/generated/ondemand/JtedirectoriesGenerated.java:7: error: cannot find symbol
                var before = directories.listMetadata.before;;
                ^
  symbol:   class var
  location: class JtedirectoriesGenerated
/Users/adamwolfman/Documents/example_apps/java-example-applications/java-directory-sync-example/jte-classes/gg/jte/generated/ondemand/JtedirectoriesGenerated.java:9: error: cannot find symbol
                var after = directories.listMetadata.after;;
                ^
  symbol:   class var
  location: class JtedirectoriesGenerated
/Users/adamwolfman/Documents/example_apps/java-example-applications/java-directory-sync-example/jte-classes/gg/jte/generated/ondemand/JtedirectoriesGenerated.java:19: error: cannot find symbol
                for (var directory : directories.data) {
                     ^
  symbol:   class var
  location: class JtedirectoriesGenerated
3 errors
```

Reply:
It runs as expected on main though, so it's gotta have something to do with this JDK version since that's all that was changed.

Reply:
Interesting that it compiles just fine... I'll just set the version to 11 since we don't really _need_ to use 8 for the example apps.

Reply:
Updated.

Reply:
Looks good now!